include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -632,12 +632,12 @@ struct __parallel_radix_sort_iteration
     {
         using _CustomName = typename __decay_t<_ExecutionPolicy>::kernel_name;
         using _RadixCountKernel =
-            __internal::__kernel_name_generator<__count_phase, _CustomName, __decay_t<_InRange>, __decay_t<_TmpBuf>>;
+            __internal::__kernel_name_generator<__count_phase, _CustomName, _ExecutionPolicy __decay_t<_InRange>, __decay_t<_TmpBuf>>;
         using _RadixLocalScanKernel =
-            __internal::__kernel_name_generator<__local_scan_phase, _CustomName, __decay_t<_TmpBuf>>;
-        using _RadixReorderPeerKernel = __internal::__kernel_name_generator<__reorder_peer_phase, _CustomName,
+            __internal::__kernel_name_generator<__local_scan_phase, _CustomName, _ExecutionPolicy __decay_t<_TmpBuf>>;
+        using _RadixReorderPeerKernel = __internal::__kernel_name_generator<__reorder_peer_phase, _CustomName, _ExecutionPolicy
                                                                             __decay_t<_InRange>, __decay_t<_OutRange>>;
-        using _RadixReorderKernel = __internal::__kernel_name_generator<__reorder_phase, _CustomName,
+        using _RadixReorderKernel = __internal::__kernel_name_generator<__reorder_phase, _CustomName, _ExecutionPolicy
```

Comment:
Looks like these are missing commas, which are causing the compilation errors.

Reply:
Thanks, fixed.